server/forge/forgejo/helper.go

```diff
@@ -155,17 +164,17 @@ func pipelineFromPullRequest(hook *pullRequestHook) *model.Pipeline {
 	}
 
 	pipeline := &model.Pipeline{
-		Event:    event,
-		Commit:   hook.PullRequest.Head.Sha,
-		ForgeURL: hook.PullRequest.HTMLURL,
-		Ref:      fmt.Sprintf("refs/pull/%d/head", hook.Number),
-		Branch:   hook.PullRequest.Base.Ref,
-		Message:  hook.PullRequest.Title,
-		Author:   hook.PullRequest.Poster.UserName,
-		Avatar:   avatar,
-		Sender:   hook.Sender.UserName,
-		Email:    hook.Sender.Email,
-		Title:    hook.PullRequest.Title,
+		Event:              event,
+		Commit:             hook.PullRequest.Head.Sha,
+		ForgeURL:           hook.PullRequest.HTMLURL,
+		Ref:                fmt.Sprintf("refs/pull/%d/head", hook.Number),
+		Branch:             hook.PullRequest.Base.Ref,
+		Author:             hook.PullRequest.Poster.UserName,
+		Avatar:             avatar,
+		Sender:             hook.Sender.UserName,
+		Email:              hook.Sender.Email,
+		PRTitleDescription: hook.PullRequest.Title + "\n" + hook.PullRequest.Body,
+		Message:            "", // TODO: get message from last commit to pr
```

Comment:
Does someone know if we can this data from the webhook somehow, or do we need to query the api again?